boards/sensry/ganymed_bob/doc/index.rst

```diff
@@ -0,0 +1,141 @@
+.. _ganymed_bob:
+
+Ganymed BOB
+############
+
+Overview
+********
+
+.. note::
+
+   All software for the nGanymed BOB is experimental and hardware availability
+   is restricted to the participants in the limited sampling program.
```

Comment:
this looks like you've taken the document from a nordic board, unless this is true, it should be removed

Reply:
yes, its taken from nordic template. its by intention. we start to introduce the chip.